test/paper-dialog-behavior.html

```diff
@@ -359,8 +354,6 @@ <h2 id="header">Dialog</h2>
               // causing a "Maximum call stack size exceeded" error.
               // Wait 50ms and verify this does not happen.
               Polymer.Base.async(function() {
-                // Should not enter in an infinite loop.
-                assert.equal(spy.callCount, 1, 'focused only once');
```

Comment:
In IE10 & IE11 the `dialogs[1].focus()` method gets called twice, while in the other browsers only once https://travis-ci.org/PolymerElements/paper-dialog-behavior/builds/118046394
The goal of this test is to just avoid `Maximum call stack size exceeded` to happen. 
I feel asserting that the `callCount <= 2 ` doesn't look that good, also that's not the goal of the test to check `focus changed only once`. WDYT?